src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/finalclass/Example1.java

```diff
@@ -6,43 +6,53 @@
 </module>
 */
 
-// xdoc section -- start
-final class Example1 {
-  private Example1() { }
-}
+package com.puppycrawl.tools.checkstyle.checks.design.finalclass;
 
-class A { // violation, 'Class A should be declared as final.'
-  private A() { }
-}
+// xdoc section -- start
+public class Example1 { // ok, since it has a public constructor
```

Comment:
It seems like all these changes in `minor: add check for package name` should actually be in `Enable examples tests for SuperCloneCheck `.

Reply:
No, these changes are required for the file to compile after adding the package declaration, and blocked compilation of other files with similarly named classes.